packages/syft/src/syft/service/worker/worker_pool_service.py

```diff
@@ -69,6 +69,7 @@ def launch(
         num_workers: int,
         registry_username: str | None = None,
         registry_password: str | None = None,
+        pod_annotations: dict[str, str] | None = None,
     ) -> list[ContainerSpawnStatus] | SyftError:
```

Comment:
can we add pod labels as well? label selectors may also used to place certain pods in certain node pools

Reply:
Addressed in [9692b6e](https://github.com/OpenMined/PySyft/pull/8884/commits/9692b6e8cbad38bf272f5e66761b6a95b1337e94)